packages/http-client/src/common-client-options.ts

```diff
@@ -0,0 +1,90 @@
+import { JSONSchemaType } from "@typespec/compiler";
+
+export interface ClientEmitterOptions {
+  "api-version"?: string;
+  "examples-dir"?: string;
+  "package-version"?: string;
+  "package-name"?: string;
+  namespace?: string;
+  license?: {
+    name: string;
+    company?: string;
+    link?: string;
+    header?: string;
+    description?: string;
+  };
+}
+
+export const ClientEmitterOptionsSchema: JSONSchemaType<ClientEmitterOptions> = {
+  type: "object",
+  additionalProperties: false,
+  properties: {
+    "package-version": {
+      type: "string",
+      nullable: true,
+      default: "0.0.1",
+      description: "The version of the package.",
+    },
+    "package-name": {
+      type: "string",
+      nullable: true,
+      default: "test-package",
+      description: "Name of the package.",
+    },
+    "examples-dir": {
```

Comment:
I thikn from the discussion on teh tcgc pr some of those are still azure only.

Reply:
Makes sense, I'll keep them all in the js emitter for now. I'll talk with the crew to see if we can have unbranded options exported from http-client and consumed by TCGC.

cc:/ @iscai-msft @tadelesh